django/contrib/admin/views/main.py

```diff
@@ -48,6 +48,7 @@
     IS_FACETS_VAR,
     IS_POPUP_VAR,
     TO_FIELD_VAR,
+    PAGE_VAR,
```

Comment:
Adding `PAGE_VAR` to `IGNORED_PARAMS` looks like on unrelated bugfix :thinking: Can you move it to the separate PR with a regression test?

Reply:
Ah no sorry it's because things get deleted from `self.params` but not from the new `self.filter_params` so `PAGE_VAR` and `ERROR_VAR` shouldn't be needed in the `IGNORED_PARAMS` - I will push up a change

Reply:
Updated [1e05ab9](https://github.com/django/django/pull/16621/commits/1e05ab98edf07f060c09f5d5c81738f7cc597183) :+1: